script/playercomponent.js

```diff
@@ -392,5 +394,13 @@ define(
         tearDown: tearDown
       };
     };
+
+    function getLiveSupport (device) {
+      return PlaybackStrategy.getLiveSupport(device);
+    }
+
+    PlayerComponent.getLiveSupport = getLiveSupport;
```

Comment:
Can this not be added on to the object like all the other public functions?

Reply:
No because tap-static needs to use it before Bigscreen Player is initialised.